wikiViewer.js

```diff
@@ -0,0 +1,64 @@
+//opening the page
+$(document).ready(function() {
+
+    //clicking on search
+
+    $('#searchButton').click(function() {
+
+
+        //define variables
+
+        var searchTerm = $('#searchTerm').val();
+
+        //API url with serach implement
+
+        var url = "https://en.wikipedia.org/w/api.php?action=opensearch&search=" + searchTerm + "&format=json&callback=?";

```

Comment:
+ wlasciwie wszedl juz es6 wszedzie (a jesli nie to uzywamy babela do konwersji do es5), wiec moglabys skorzystac z https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/template_strings